src/sth_helper.js

```diff
@@ -117,9 +117,9 @@
       return sthConfig.UNICA_CORRELATOR.NOT_AVAILABLE;
     } else {
       return getHashCode('from: ' + request.info.remoteAddress +
-                         ':' + request.info.remotePort +
-                         ', method: ' + request.method.toUpperCase() +
-                         ', url: ' + request.url.path);
+      ':' + request.info.remotePort +
```

Comment:
What does the checkstyle XML say about this indentation? Java-related one would say this is not correct. In adition, it would say the `+` opertor must start the new line... but that are the Java guidelines :)